### PR TITLE
WIP: Onboard ObservabilityAddon resource

### PR DESCRIPTION
**Pull request by omrirh**

### CODE REVIEW

File: ocp_resources/resource.py [hunk 1]
```diff
@@ -317,6 +317,9 @@ class ApiGroup:
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VELERO_IO = "velero.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        OBSERVABILITY_ADDON_IO = (
```
Comment (myakove) [inline ocp_resources/resource.py:320]: This should be sorted by ABC

File: ocp_resources/observability_addon.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class ObservabilityAddon(NamespacedResource):
```
Comment (myakove) [inline ocp_resources/observability_addon.py:4]: Missing call to super and implementation on to_dict

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/observability_addon.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    """
+    https://github.com/stolostron/multicluster-observability-operator/blob/main/docs/MultiClusterObservability-CRD.md
```
Comment (rnetser) [inline ocp_resources/observability_addon.py:6]: is there an api doc?

File: ocp_resources/resource.py (hunk 1 above)
Comment (rnetser) [inline ocp_resources/resource.py:320]: use `OBSERVABILITY_OPEN_CLUSTER_MANAGEMENT_IO` in f-string

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/observability_addon.py (hunk 3 above)
Comment (omrirh) [reply ocp_resources/observability_addon.py:6]: haven't found one yet; will update once I do

File: ocp_resources/resource.py (hunk 1 above)
Comment (omrirh) [reply ocp_resources/resource.py:320]: done

File: ocp_resources/observability_addon.py (hunk 2 above)
Comment (omrirh) [reply ocp_resources/observability_addon.py:4]: I wish to use this resource for view only after the observability is enabled.
Should we implement a to_dict anyway?

Comment (myakove) [reply ocp_resources/observability_addon.py:4]: Yes, this is not only for your personal use, this should be full resource implementation

**Pull request closed without merging**